docs/team/alanlowzies.md

```diff
@@ -1,6 +1,103 @@
-# Alan - Project Portfolio Page
+# Project Portfolio Page of Alan Low (alanlowzies)
 
 ## Overview
+This page showcases my contributions to the development of WerkIt!, a team project (tP) in the CS2113T 
+Software Engineering & Object-Oriented Programming module offered by the School of Computing, National University of 
+Singapore.
 
+### About the Project
+WerkIt! is a command line interface (CLI) application written in Java hat allows users to create a weekly workout 
+schedule for them to refer to and follow. More details about the project can be found in the following locations:
+* [GitHub Repository](../../)
+* [WerkIt! User Guide](../UserGuide.md)
+* [WerkIt! Developer Guide](../DeveloperGuide.md)
 
-### Summary of Contributions
+---
+
+## Summary of Contributions
+### Code Contributed
+A detailed report of my code contributions to the tP can be found in the [tP Code Dashboard](https://nus-cs2113-ay2122s2.github.io/tp-dashboard/?search=alanlowzies&breakdown=true) 
+hosted by the module coordinators of CS2113T.
+
+![tP Code Dashboard](../images/ppp/alanlowzies/tPCodeDashboard.png)
+
+A summary of my code contributions are as follows:
+- Wrote most of the `FileManager` API, which allows WerkIt! to read and write application data
+into text files stored on the user's local filesystem 
+([[WIP] Developer Guide]()). 
+- Implemented the functionality for users to create a new workout 
+([User Guide](../UserGuide.md#create-a-workout-workout-new) | [Developer Guide](../DeveloperGuide.md#create-new-workout)).
+- Wrote most of the `UI` API, which contains UI-related elements, including WerkIt!'s banner logo and response messages.
+- Wrote most of the `WerkIt` API, which contains the code to start up WerkIt!, load its components, load saved file data
+  (if any), and finally receive and process user inputs until the user exits the program.
+- Wrote parts the `Parser` API, which parses user inputs into the appropriate `Command` class types as well
+  as check for invalid characters in the user inputs ([Developer Guide](../DeveloperGuide.md#parsing-user-input-and-getting-the-right-command)).
+- Implemented the HashMap data structure in the `WorkoutList` API to allow `PlanList` to reference
+  workouts created by the user ([Developer Guide](../DeveloperGuide.md#design-considerations-for-creating-a-new-workout)).
+- Wrote the `LogHandler` API, which allows the various components in WerkIt! to log into a file
+  stored in the user's local filesystem ([[WIP] Developer Guide]()).
+- Wrote the `Command` abstract class, which is a template class that sets out the main structure of all types
+  of commands in WerkIt! (e.g. `WorkoutCommand`, `ExerciseCommand`, etc.)
+- Wrote most of the `WorkoutCommand` API, which contains a workout-related command that the user has requested
+  WerkIt! to carry out.
+- Wrote some of the exception APIs for WerkIt!, including `InvalidWorkoutException`, `InvalidExcerciseException`, and
+  `UnknownFileException`.
```

Comment:
Typo on `InvalidExerciseException`

Reply:
Good spot! Thanks for finding it :) Fixed in Commit c71b98c.